public/docs/_examples/toh-5/dart/lib/app_component_1.dart

```diff
@@ -1,25 +1,31 @@
 // #docplaster
-// #docregion
+// #docregion , v2
 import 'package:angular2/core.dart';
-// #enddocregion
-import 'package:angular2/router.dart'; // for testing only
-// #docregion
 
 import 'hero_service.dart';
 import 'heroes_component.dart';
 
+// #enddocregion v2
 @Component(
```

Comment:
I wondered about having two `@Component` annotations for a single class. Sure enough, when I try to `pub serve` or `pub build` this, I get a build error:

```
Build error:
Transform DirectiveProcessor on angular2_tour_of_heroes|lib/app_component_1.dart threw error: Only one Directive is allowed per class. Found unexpected "@Component(selector: 'my-app', template: '''
      <h1>{{title}}</h1>
      <a [routerLink]="['Heroes']">Heroes</a>
      <router-outlet></router-outlet>''', directives: const [HeroesComponent], providers: const [HeroService])".
@Component(selector: 'my-app', template: '''
      <h1>{{title}}</h1>
     ...
```

Does the serve/build work without errors for you?